tests/test_two_sided_market.py

```diff
@@ -225,8 +225,8 @@ def test_double_sided_pay_as_clear_market_works_with_floats(self, pac_market):
             Bid("bid_id2", pendulum.now(), 2.2, 1, "B", buyer_origin="S").serializable_dict(),
             Bid("bid_id3", pendulum.now(), 1.1, 1, "B", buyer_origin="S").serializable_dict()]
 
-        matched = pac_market.get_clearing_point(bids, offers, pendulum.now())[0]
-        assert matched == 2.2
+        matched = pac_market.get_clearing_point(bids, offers, now(), str(uuid4()))
+        assert matched[0] == 2.2
```

Comment:
Nit: For better understanding:
```suggestion
        clearing_rate, _ = pac_market.get_clearing_point(bids, offers, now(), str(uuid4()))
        assert clearing_rate == 2.2
```